src/client/containers/PullRequestIssues/__tests__/index.js

```diff
@@ -0,0 +1,100 @@
+import chai from 'chai'
+import {
+  getData,
+} from '../index'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getData', () => {
```

Comment:
Same here, name it correctly, otherwise, it will appear two identical tests in test repost (as with PullRequestDiscussion) and someone will have hard time to figure out which one is failed